app/controllers/users/sessions_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Users::SessionsController < Devise::SessionsController
```

Comment:
Style/ClassAndModuleChildren: Use nested module/class definitions instead of compact style.